src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Unix/PkcsFormatReader.cs

```diff
@@ -38,6 +49,11 @@ internal static bool IsPkcs7Der(SafeBioHandle fileBio)
         {
             using (SafePkcs7Handle pkcs7 = Interop.Crypto.D2IPkcs7Bio(fileBio))
             {
+                if (pkcs7.IsInvalid)
+                {
+                    Interop.Crypto.ErrClearError();
+                }
+
                 return !pkcs7.IsInvalid;
```

Comment:
Nit: This is inconsistent from other similar cases.  To be consistent it would be:
```C#
if (pkcs7.IsInvalid)
{
    Interop.Crypto.ErrClearError();
    return false;
}

return true;
```